member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java

```diff
@@ -79,7 +80,14 @@ public MemberAggregate(CreateMemberCommand command) {
           return new AcceptLanguageUpdatedEvent(command.getMemberId(), command.getAcceptLanguage());
         }
         return null;
-        });
+        })
+      .andThenApply(() ->
+      {
+        if(command.getMarket() != null && !command.getMarket().toString().isEmpty()){
```

Comment:
Why would the string be empty in an enum?
```suggestion
        if(command.getMarket() != null){
```

Reply:
👍  Legacy from when it was a string and not enum.